decidim-budgets/spec/types/integration_schema_spec.rb

```diff
@@ -79,13 +87,21 @@
           "totalCommentsCount" => project.comments_count,
           "type" => "Decidim::Budgets::Project",
           "updatedAt" => project.updated_at.to_time.iso8601,
-          "userAllowedToComment" => project.user_allowed_to_comment?(current_user)
+          "userAllowedToComment" => project.user_allowed_to_comment?(current_user),
+          "followsCount" => project.follows_count,
```

Comment:
here too, we should follow alphabetical order

Reply:
done in 9ebeb0b8a9